### DISCUSSION TITLE
v1.5.3-beta

### DISCUSSION THREAD

**Discussion by albertpb** [albertpb/sd-manager#17] [General]
- Fix stuck at generating thumbnails
- Fix wrong rating value
 

<hr /><em>This discussion was created from the release <a href='https://github.com/albertpb/sd-manager/releases/tag/v1.5.3'>v1.5.3-beta</a>.</em>

**Comment by AsuraAce** [comment]
- thumbnails seem to work, not stuck anymore (still running)
- is there a fallback/recovery for deleted thumbnails? it should regenerate missing ones when updating the image DB in SD-Manager. it didn't yesterday, which resulted in "unknown" thumbnails.
 
- a more efficient update method would be nice. lets say i forgot to start SD-Manager and i want to get those images i generated into the DB, as it stands now (it seems) i would have to run an update that takes like 10-15 minutes.
- an option to add thumbnails from external images and or images from within SD-Manager would be nice as well. useful for merged models and models SDM couldn't find / update.

**Comment by AsuraAce** [comment]
- "image animated": not working, didn't work with prior versions all the time either. when it works it sometimes misses images.
-  not all images are loaded / found. recent images with certain models like Dreamshaper 8 are not loaded. not in the images tab, checkpoint tab nor the lora tab. SDM couldn´t find any images for some of the models.

![image](https://github.com/albertpb/sd-manager/assets/13394741/5ad0f93f-5b18-47b6-be93-52c3b3e82d37)

**Reply by albertpb** [reply]
I'm only filtering if are .png files.  can you post an image so I can test ?

**Comment by AsuraAce** [comment]
lora and checkpoints have a 5 star rating per default (only in the overview, not in the model view itself)

ignore the first one, this one i have rated myself. seems to only be a visual a thing. 
would probably keep the 5 stars rating as a default but change the color

![image](https://github.com/albertpb/sd-manager/assets/13394741/c8240ffb-2674-4b40-bd0c-f50b14551c06)

**Comment by AsuraAce** [comment]
drag & drop from the view below uses the thumbnail (i think) and is  lacking generation / metadata (probably because it is the thumbnail). drag and drop everywhere else (?) is not working

![image](https://github.com/albertpb/sd-manager/assets/13394741/c32b1ef8-a320-4ff4-be7f-b3b74c434818)

**Comment by AsuraAce** [comment]
"update images database" with the button provided in the image tab, doesn't work. you get stuck at 0.00%

![image](https://github.com/albertpb/sd-manager/assets/13394741/32e4cc34-c963-4374-8ce3-2e08b7dd38b8)

![image](https://github.com/albertpb/sd-manager/assets/13394741/d34d180d-0e81-4c05-83d1-200e7e5c87ae)

**Reply by albertpb** [reply]
does it happens when you import in the settings ?

**Reply by AsuraAce** [reply]
works fine from the settings.

![image](https://github.com/albertpb/sd-manager/assets/13394741/43c8545b-05b9-4f08-afbc-3f1be5a1472f)
